source/_components/switch.template.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: component
+title: "Template switch"
+description: "Instructions how to integrate Template switches into Home Assistant."
+date: 2016-02-06 07:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Switch
+---
+
+The `template` platform creates switches that combine other components.
```

Comment:
We could have some more explanation here how other components are actually combined.